### DISCUSSION TITLE
Segmentation from Human

### DISCUSSION THREAD

**Discussion by YaraAmin** [vladmandic/human#169] [Q&A] [answered]
Is there any way to set config to human so I can get the person segmentation something like the output of BodyPix? 
or do I have to overwrite this https://github.com/vladmandic/human/blob/66428abc122fac1c003fc30f6d3914414c181b6c/src/segmentation/segmentation.ts#L45 ?

**Comment by vladmandic** [comment, marked as answer]
body segmentation functionality has been mostly just a prototype  
i've given it some attention today and just pushed an update on github  

should be faster, more precise and configurable  
plus main demo app is updated to clearly visualize whats going on when segmentation is enabled  
and outputs are usable other than just replacing current canvas in-place  

see docs at:
- <https://vladmandic.github.io/human/typedoc/classes/Human.html#segmentation>
- <https://vladmandic.github.io/human/typedoc/interfaces/SegmentationConfig.html>

hope this helps, but let me know what else you're interested in

**Comment by YaraAmin** [comment]
Thank you so much for spending time on that. That's really helpful.
I will just need to install Human again and all updates will be added, right?

**Comment by vladmandic** [comment]
if you're using a package, i haven't published a new version on `npmjs` yet but you can install from `github` main branch directly:  
> npm install git+https://github.com/vladmandic/human

new version will likely be published on `npmjs` on monday

**Comment by YaraAmin** [comment]
Thank you

**Reply by vladmandic** [reply]
Btw, I just published v2.2.3 ahead of original schedule.
